tests/foreman/cli/test_variables.py

```diff
@@ -283,7 +288,8 @@ def test_positive_update_variable_puppet_class(self):
         smart_variable = make_smart_variable({
             'puppet-class': self.puppet['name']})
         self.assertEqual(smart_variable['puppet-class'], self.puppet['name'])
-        new_puppet = Puppet.info({u'name': 'ntp::config'})
+        new_puppet = Puppet.info(
+            {u'name': self.puppet_modules[1]['name']})
```

Comment:
@abalakh Well, the purpose of this is to take any other class besides the default one and it doesn't depend whether this class was used by other tests. Pop can cause troubles in case someone accidentally removes second module from the list.